src/Tickets/Emails/JSON_LD/Handler.php

```diff
@@ -0,0 +1,103 @@
+<?php
+namespace TEC\Tickets\Emails\JSON_LD;
+
+use TEC\Tickets\Commerce\Module;
+use TEC\Tickets\Emails\Email_Abstract;
+
+/**
+ * Class Handler
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Emails\JSON_LD
+ */
+class Handler {
+
+	/**
+	 * Get the JSON LD data for the email.
+	 *
+	 * @since TBD
+	 *
+	 * @param Email_Abstract $email The email object.
+	 *
+	 * @return array The JSON LD data.
+	 */
+	public static function get_data( \TEC\Tickets\Emails\Email_Abstract $email ): array {
+
+		$data = [];
+
+		if ( $email->slug === 'completed-order'
+		     || $email->slug === 'purchase-receipt' )
+		{
+			$order = $email->__get( 'order' );
+
+			if ( empty( $order ) ) {
+				return [];
+			}
+
+			return ( new Handler )->get_completed_order_data( $order );
+		}
+
+		return $data;
+	}
+
+	/**
+	 * Get the JSON LD data for a completed order.
+	 *
+	 * @since TBD
+	 *
+	 * @param $order \WP_Post The order post object.
+	 *
+	 * @return array
+	 */
+	public function get_completed_order_data( $order ) : array {
```

Comment:
```suggestion
	public function get_completed_order_data( $order ): array {
```

Reply:
Yes, that's the ultimate plan. For now, it's just all in one place for focus, once we have all the data types we can then distribute them into separate classes.